skbase/utils/dependencies/tests/test_check_dependencies.py

```diff
@@ -47,3 +49,53 @@ def test_check_soft_deps():
         assert _check_soft_dependencies(
             ("pytest", "!!numpy<~><>0.1.0"), severity="none"
         )
+
+
+@patch("sktime.utils.dependencies._dependencies.sys")
```

Comment:
causes error, should be `skbase`

Reply:
you are right sorry, classical case of paste and not edit...
I just copied the whole test from sktime repo and forgot to change it...